lib/ssz_ex.ex

```diff
@@ -0,0 +1,36 @@
+defmodule LambdaEthereumConsensus.SszEx do
+  @moduledoc """
+    SSZ library in Elixir
+  """
+  #################
+  ### Public API
+  #################
+  def encode(value, {:int, size}), do: encode_int(value, size)
+  def encode(value, :bool), do: encode_bool(value)
+
+  def decode(binary, :bool), do: decode_bool(binary)
+  def decode(binary, {:int, size}), do: decode_uint(binary, size)
+
+  #################
+  ### Private functions
+  #################
+  defp encode_int(value, size) when is_integer(value) do
+    <<encoded::binary-size(div(size, 8)), _rest::binary>> =
+      value
+      |> :binary.encode_unsigned(:little)
+      |> String.pad_trailing(div(size, 8), <<0>>)
+
+    encoded
```

Comment:
I think this can be simplified to:

```elixir
defp encode_int(value, size) when is_integer(value), do: <<value::size(size)-little>>
```

Or am I missing something?

Reply:
it is necessary to add  ``<<0>>`` depending of the size. 
For example `5` with size u8 is ``<<5>>``, but if the size is u16 the encoding should be ``<<5, 0>>`` and so on.

Reply:
oh!! you are right, like magic